model/marketplace_plugin.go

```diff
@@ -14,18 +14,26 @@ import (
 	"github.com/pkg/errors"
 )
 
-// BaseMarketplacePlugin is a Mattermost plugin received from the marketplace server.
+// BaseMarketplacePlugin is a Mattermost plugin received from the Marketplace server.
 type BaseMarketplacePlugin struct {
-	HomepageURL     string `json:"homepage_url"`
-	IconData        string `json:"icon_data"`
-	DownloadURL     string `json:"download_url"`
-	ReleaseNotesURL string `json:"release_notes_url"`
-	// Signature represents a signature of a plugin saved in base64 encoding.
-	Signature string    `json:"signature"`
-	Manifest  *Manifest `json:"manifest"`
+	HomepageURL     string             `json:"homepage_url"`
+	IconData        string             `json:"icon_data"`
+	DownloadURL     string             `json:"download_url"`
+	ReleaseNotesURL string             `json:"release_notes_url"`
+	Labels          []MarketplaceLabel `json:"labels"`
```

Comment:
this translates to the following json, yes?
```
{
...
  "labels": [],
...
}
```

Reply:
I'm not sure I get the question. If no labels are provided, it will be an empty list.

Reply:
👌 i think if this was `[]*MarketplaceLabel` then it would translate to `"
labels": null` in JSON, so was wondering if we needed that distinction. But `[]MarketplaceLabel` should work just fine.